.github/workflows/run_tests.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: Install SmartSim (with ML backends)
         run: |
           python -m pip install git+https://github.com/CrayLabs/SmartRedis.git@develop#egg=smartredis
-          python -m pip install .[dev,ml]
+          python -m pip install .[dev,mypy,ml]
```

Comment:
Is this a change you want to add now or in one of your upcoming PRs?

Reply:
good catch. i'm going to revert and percolate code from PRs 652/653 into my feature